app/src/main/java/com/example/news_app/PersonalInformationActivity.java

```diff
@@ -1,67 +1,167 @@
 package com.example.news_app;
 
-import androidx.activity.result.ActivityResultCallback;
-import androidx.activity.result.ActivityResultLauncher;
-import androidx.activity.result.contract.ActivityResultContracts;
-import androidx.annotation.Nullable;
-import androidx.appcompat.app.AppCompatActivity;
-
+import android.annotation.SuppressLint;
+import android.app.ProgressDialog;
 import android.content.Intent;
 import android.net.Uri;
 import android.os.Bundle;
-import android.view.View;
+import android.util.Log;
 import android.widget.Button;
 import android.widget.ImageView;
 import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.activity.result.ActivityResultLauncher;
+import androidx.activity.result.contract.ActivityResultContracts;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
 
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.resource.bitmap.CenterCrop;
+import com.bumptech.glide.load.resource.bitmap.CenterInside;
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners;
 import com.example.news_app.model.Parse.User;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.storage.FirebaseStorage;
+import com.google.firebase.storage.StorageReference;
 import com.parse.ParseUser;
 
+import java.util.HashMap;
+
 public class PersonalInformationActivity extends AppCompatActivity {
 
+    public static final String TAG = "PERSONAL INFORMATION ACTIVITY";
+    private ProgressDialog progressDialog;
+    private String photoUrl;
     private User currentUser;
     private TextView tv_first_name;
     private TextView tv_last_name;
     private TextView tv_user_name;
     private ImageView iv_profile_pic;
     private Button btn_change_picture;
+    private Button btn_save;
+    private Uri photoUri;
+    private String photoFileName = "photo.jpg";
```

Comment:
where is this used? if it is always photo.jpg, then it should be a static final value in all caps like line 35.  is it ways photo.jpg?